declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,63 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="customizing-permissions-by-creating-aggregated-cluster-roles"]
+= Customizing permissions by creating aggregated cluster roles
+:context: customizing-permissions-by-creating-aggregated-cluster-roles
+
+toc::[]
+
+The default cluster role for the Argo CD Application Controller has a specific set of hard-coded permissions. The {gitops-title} Operator manages this cluster role, so you cannot modify it. As a cluster administrator, you can customize the permissions by using any one of the following methods:
```

Comment:
Suggestion: The {gitops-title} Operator is managing this cluster role, so you cannot modify it.

Reply:
Discussed with the SME and sorted the conversation with @Dhruv-Soni11. Retaining the content as is.